_data/simple-icons.json

```diff
@@ -5565,7 +5569,7 @@
         {
             "title": "MetaFilter",
             "hex": "065A8F",
-            "source": "https://www.metafilter.com/apple-touch-icon.png"
+            "source": "https://www.metafilter.com/search.mefi?site=mefi&q=logo"
```

Comment:
The original source can be found in their homepage source.

Reply:
where in the homepage is the icon?

Reply:
as favicon, I see.